CHANGELOG.md

```diff
@@ -18,6 +18,8 @@
 - [#7177](https://github.com/influxdata/influxdb/issues/7177): Fix base64 encoding issue with /debug/vars stats.
 - [#7196](https://github.com/influxdata/influxdb/issues/7196): Fix mmap dereferencing, fixes #7183, #7180
 - [#7013](https://github.com/influxdata/influxdb/issues/7013): Fix the dollar sign so it properly handles reserved keywords.
+- [#7196](https://github.com/influxdata/influxdb/ssues/7196): Fix mmap dereferencing, fixes #7183, #7180
+- [#2792](https://github.com/influxdata/influxdb/ssues/2792): Exceeding max retention policy duration gives incorrect error message
```

Comment:
I fixed this typo in a PR so be careful not to reintroduce it when you rebase. You also have `ssues` instead of `issues` here.